### DISCUSSION TITLE
Quick hide button on tile?

### DISCUSSION THREAD

**Discussion by Omie** [dailydotdev/daily#583] [Feature Requests]
Hi

I was wondering if it's possible to get just the **Hide** button out on the main tile. 

Even though I have selected sources I am very interested in there are usually articles I'd like to quickly hide for number of reasons. Most of the times I have read them before from other sources (HN/Twitter etc), sometimes the title is sufficient to know what's inside, sometimes it's evident that it's too basic for my help etc.

Thank you

**Comment by idoshamun** [comment]
Hi @Omie,

We have a "hide" button in the options menu of the card.
Please see the screenshot below and let me know if it addresses your request.
<img width="362" alt="image" src="https://user-images.githubusercontent.com/1993245/173742132-06817908-5ddd-4ca0-8bdd-0361a0a26a89.png">

**Reply by Omie** [reply]
it's 2 clicks far, I was curious if it can be a single click workflow ;-)

**Reply by idoshamun** [reply]
The card is already pretty crowded. So we are very picky on what gets to the frontline

**Reply by Omie** [reply]
alright, makes sense. Thanks
